operators/benchmark-receiver/run.py

```diff
@@ -0,0 +1,21 @@
+from typing import Any
+
+from interactem.core.logger import get_logger
+from interactem.core.models.messages import BytesMessage
+from interactem.operators.operator import operator
+
+logger = get_logger()
+
+@operator
+def receive_benchmark_frame(
+    inputs: BytesMessage | None, parameters: dict[str, Any]
+) -> None:
+
+    if inputs:
+        meta = inputs.header.meta if hasattr(inputs.header, 'meta') else {}
```

Comment:
if inputs is not None, we don't have to do this `if hasattr(inputs.header, 'meta')` since `BytesMessage` has a dict as its default value. 

We should avoid running unnecessary `if` statements because that will impact performance.